### DISCUSSION TITLE
Source code of the PDEs and ODEs solved in JULES

### DISCUSSION THREAD

**Discussion by CorentinHoupert** [MetOffice/simulation-systems#108] [JULES]
Hello,

I would like to know how the PDEs and ODEs in JULES are solved.

I already know some equations are stated in these papers of 2011 and 2015
https://gmd.copernicus.org/articles/4/677/2011/gmd-4-677-2011.pdf
https://gmd.copernicus.org/articles/4/701/2011/gmd-4-701-2011.pdf
https://gmd.copernicus.org/articles/8/1139/2015/gmd-8-1139-2015.pdf

Do you know which part of the source code corresponds to it?
Do you know how can I have access to it?

Best,
Corentin

**Comment by doucla** [comment]
Corentin,

There's no short answer to this, not least because the documentation papers cover most aspects of the model, and obviously the source code covers it all. Are there particular aspects of the model that you are most interested in?

In any case you will probably want to look at the source code. For access you will need to register - see this form: https://jules-lsm.github.io/access_req/JULES_access.html

Doug
